packages/circle-demo-webapp/app/routes/wallet.$id/components/WalletAddressDialog.tsx

```diff
@@ -4,25 +4,25 @@ import { Wallet } from 'web3-circle-sdk';
 
 import { Button } from '~/components/ui/button';
 import { Dialog, DialogContent, DialogTrigger } from '~/components/ui/dialog';
-import { WalletReceive } from '~/components/WalletReceive';
+import { WalletAddressHelpers } from '~/components/WalletAddressHelpers';
 
-interface WalletReceiveDialogProps {
+interface WalletAddressDialogProps {
   wallet: Wallet;
 }
 
-export function WalletReceiveDialog({ wallet }: WalletReceiveDialogProps) {
+export function WalletAddressDialog({ wallet }: WalletAddressDialogProps) {
   const [open, setOpen] = useState(false);
 
   return (
     <Dialog open={open} onOpenChange={setOpen}>
       <DialogTrigger asChild>
         <Button variant="outline">
-          <ArrowDown /> Receive
+          <ArrowDown /> Address
```

Comment:
The button Receive is adapted from the design so I'd not change that 
<img width="307" alt="Screenshot 2024-12-13 at 09 38 36" src="https://github.com/user-attachments/assets/a584f123-e64c-4509-a781-ec6b3ed432e2" />